packages/auth/src/core/credentials/email.ts

```diff
@@ -32,7 +32,11 @@ import { AuthErrorCode } from '../errors';
 import { _fail } from '../util/assert';
 import { AuthCredential } from './auth_credential';
 import { handleRecaptchaFlow } from '../../platform_browser/recaptcha/recaptcha_enterprise_verifier';
-import { RecaptchaActionName, RecaptchaClientType } from '../../api';
+import {
+  RecaptchaActionName,
+  RecaptchaClientType,
+  RecaptchaProvider
```

Comment:
`RecaptchaAuthProvider`? `RecaptchaProvider` sounds like providers of recaptcha capability itself.

Reply:
Done. Changed to `RecaptchaAuthProvider`.